public/maintenance.html

```diff
@@ -48,9 +48,8 @@
       <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
         <div class="govuk-grid-row">
           <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-l">Sorry, the service is unavailable</h1>
-            <p class="govuk-body">You will be able to use the service from 8:30pm.</p>
-            <p class="govuk-body">If you are part way through a check, we saved your answers. They will be available for 14 days from when you started the check.</p>
+            <h1 class="govuk-heading-l">Sorry, the service is unavailable because of planned maintenance</h1>
+            <p class="govuk-body">It will not be available on Wednesday 24 January 2024 from 6pm for up to 2 hours.</p>
```

Comment:
Hey @willc-work, should we call this part 1 of EL-1273? 

I guess part 2 would be to put maintenance mode to `NOT_ENABLED` and add in the generic text as [per the ticket](https://dsdmoj.atlassian.net/browse/EL-1273)?

Reply:
this only turns maintenance mode on in UAT, once it has been UAT'd that will be reverted before it is merged

Reply:
I will clarify with Laurence re: the generic content